Makefile

```diff
@@ -1,59 +1,95 @@
-RELEASE_DIR		= release/prototype
-DIAZO_DIR       = src/ploneintranet/theme/static
-LATEST          = $(shell cat prototype/LATEST)
+RELEASE_DIR	= release/prototype/_site
+DIAZO_DIR       = src/ploneintranet/theme/static/generated
+LATEST          = $(shell cat LATEST)
 BUNDLENAME      = ploneintranet
-BUNDLEURL		= https://products.syslab.com/packages/$(BUNDLENAME)/$(LATEST)/$(BUNDLENAME)-$(LATEST).tar.gz
+BUNDLEURL	= https://products.syslab.com/packages/$(BUNDLENAME)/$(LATEST)/$(BUNDLENAME)-$(LATEST).tar.gz
 
-all:: diazo
+all:: fetchrelease
 default: all
 clean:
 	rm bin/* .installed.cfg || true
 check-clean:
 	test -z "$(shell git status --porcelain)" || (git status && echo && echo "Workdir not clean." && false) && echo "Workdir clean."
 
+fetchrelease:
+	# update LATEST in case we updated the prototype
+	$(eval LATEST := $(shell cat LATEST))
+	$(eval BUNDLEURL := https://products.syslab.com/packages/$(BUNDLENAME)/$(LATEST)/$(BUNDLENAME)-$(LATEST).tar.gz)
+	echo $(BUNDLEURL)
```

Comment:
ploneintranet has it's own LATEST which is updated automatically from the prototype in the 'make prototype' step. Because the Makefile variables $(LATEST) and $(BUNDLEURL) are defined at startup time (parsing the Makefile) we need to re-set those variables.

I.e. if you run 'make diazorelease' that also updates the prototype, including the LATEST stamp there, which means $(LATEST) and $(BUNDLEURL) have to be recalculated.

There is also a 'make jsrelease'.